src/openrct2/ride/Ride.cpp

```diff
@@ -1926,6 +1914,15 @@ static void ride_music_update(Ride* ride)
     OpenRCT2::RideAudio::UpdateMusicInstance(*ride, rideCoords, sampleRate);
 }
 
+static void ride_music_update(Ride* ride)
```

Comment:
```suggestion
static void ride_music_update(Ride* ride)
```
New functions should be titlecase